bin/dmarc_send_reports

```diff
@@ -101,9 +101,9 @@ sub send_single_report {
 
     alarm($alarm_at);
 
```

Comment:
rather than 3 postfix if's, I'd rather just put this inside an `if () {` conditional.